scripts/lint-require-i18n-resource-first.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+if rg --quiet --multiline '^\(ns.*\n^\s*\(:require\n(^\s*(;|#_).*\n)*(^\s*\[status-im2\.setup\.i18n-resources\W)' src/status_im2/core.cljs; then
```

Comment:
Where exactly are you getting this `rg` from?

Reply:
Oh, I see, it was added here:
https://github.com/status-im/status-mobile/blob/45df308dd0d0e2b3945a70f514dcf5fc5f596abe/nix/shells.nix#L23-L24